src/main/resources/templates/caseoverview.html

```
@@ -0,0 +1,175 @@
+                                onclick="document.getElementById('evidenceImages').click()">
+                            Lägg till bilder
+                        </button>
+                    </div>
+                    <table>
+                        <th:block th:each="report : ${reportList}">
+                            <tr>
+                                <th class="file-name" th:text="${report.getEvent()}"></th>
+                            </tr>
+                            <tr class="table description">
+                                <td>
+                                    <span class="file-type">Beskrivning i text, eller om det är en pdf/bild -> länk</span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)